### PR TITLE
[SECURITY-1025] Remove warning from batch-task

### PR DESCRIPTION
**Pull request by lacostej**
The issue https://issues.jenkins.io/browse/SECURITY-1025

The release
* https://github.com/jenkinsci/batch-task-plugin/releases/tag/1.398.v550a_8e8ca_e5f
* https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/batch-task/1.398.v550a_8e8ca_e5f/

### CODE REVIEW

File: resources/warnings.json [hunk 1]
```diff
@@ -10557,8 +10557,8 @@
     "url": "https://www.jenkins.io/security/advisory/2022-01-12/#SECURITY-1025",
     "versions": [
       {
-        "lastVersion": "1.19",
-        "pattern": ".*"
+        "lastVersion": "1.398.v550a_8e8ca_e5f",
```
Comment (yaroslavafenkin) [inline resources/warnings.json:10560]: This should be last affected version, which seems to be 1.19 according to https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/batch-task/

File: resources/warnings.json [hunk 2]
```diff
@@ -10557,8 +10557,8 @@
...
+        "pattern": "(1[.][0-9]|1[.]1[0-9])(|[.-].+)"
```
Comment (yaroslavafenkin) [inline resources/warnings.json:10561]: Then the regex would be `(1[.]1[5-9])(|[.-].+)`

File: resources/warnings.json (hunk 1 above)
Comment (lacostej) [reply resources/warnings.json:10560]: oh well. Makes sense. I thought it was the first unaffected version :). Fixing

**Comment by Wadeck** [comment]
Thanks @lacostej for the correction 👍

**Pull request merged**